templates/sidebar/case-sidebar.html.twig

```diff
@@ -18,7 +18,7 @@
             {% block sidebarSubmenu %}
                 <div class="sidebar-sticky pt-3">
                     <h6 class="px-3 mb-1">{% trans %}Case{% endtrans %} {{ case.caseNumber }}</h6>
-                    <p class="text-muted px-3 mt-1 mb-3"><small>{{ case.complainantAddress }}</small></p>
+                    <p class="text-muted px-3 mt-1 mb-3"><small>{{ attribute(case, 'getLeaseAddress') is defined ? case.leaseAddress : case.complainantAddress }}</small></p>
```

Comment:
This can be written as `{{ case.leaseAddress|default(case.complainantAddress) }}`.

Reply:
Used `{{ case.leaseAddress|default(case.complainantAddress) }}`.